include/deal.II/matrix_free/util.h

```diff
@@ -56,6 +58,15 @@ namespace internal
               return {ReferenceCells::get_simplex<dim>(),
                       dealii::hp::QCollection<dim - 1>(
                         QWitherdenVincentSimplex<dim - 1>(i))};
+
+          for (unsigned int i = 1; i <= 3; ++i)
+            {
+              FE_SimplexP<dim> fe(i);
+              if (quad == Quadrature<dim>(fe.get_unit_support_points()))
+                return {ReferenceCells::get_simplex<dim>(),
+                        dealii::hp::QCollection<dim - 1>(Quadrature<dim - 1>(
+                          fe.get_unit_face_support_points()))};
```

Comment:
If I am not mistaken, this is empty in the DG case. Is this what you want or should it work both for CG and DG?

Reply:
I want it to be non-empty for the DG case analog to the standard `QGaussSimplex`. That's what it is doing right now, as `fe` is the CG version.

Reply:
Oh right. You are creating a dummy `FE_SimplexP`...